src/jvm/main/org/jetbrains/kotlinx/lincheck/transformation/transformers/DeterminismTransformers.kt

```diff
@@ -66,34 +72,94 @@ internal class DeterministicHashCodeTransformer(
 }
 
 /**
- * [DeterministicTimeTransformer] tracks invocations of [System.nanoTime] and [System.currentTimeMillis] methods,
+ * [FakeDeterministicTimeTransformer] tracks invocations of [System.nanoTime] and [System.currentTimeMillis] methods,
```

Comment:
Let's now split it into `DeterministicRandomTransformers.kt` and `DeterministicTimeTransformers.kt`.